src/components/Avatar.tsx

```diff
@@ -112,7 +112,7 @@ function Avatar({
                         src={avatarSource}
                         height={iconSize}
                         width={iconSize}
-                        fill={imageError ? theme.offline : iconColors?.fill ?? fill}
+                        fill={imageError ? theme.border : iconColors?.fill ?? fill}
```

Comment:
Revert. Instead update the background color returned by `getAvatarStyle`
https://github.com/Expensify/App/blob/469b11237c3079232d9bbf65899c3a4caa387b26/src/styles/utils/index.ts#L1228-L1236

Reply:
https://github.com/Expensify/App/pull/38674#issuecomment-2024192238

Reply:
It should never be the offline color

Reply:
That's the current code, not the suggestion.

Reply:
I changed this function too

Reply:
Changing the color fill color here does not fix the outlined issue but causes regressions

Reply:
Please revert. This causes a regression when offline

<img width="371" alt="Screenshot 2024-03-28 at 3 12 37 PM" src="https://github.com/Expensify/App/assets/16493223/97b332b0-78ec-4613-aa79-481c01f0a685">


The change in getAvatarStyle is enough

https://github.com/Expensify/App/assets/16493223/084b0a54-252d-4f02-8689-7771f0a164b9

Reply:
> Changing the color fill color here does not fix the outlined issue but causes regressions

hmm I don't think this line caused the regression. That's the correct fill color. But the icon should be the fallback icon right? I'll look at this today

Reply:
Sorry @s77rt could you rephrase? I'm a bit confused. The changes I made reflect this design comment https://github.com/Expensify/App/pull/38674#issuecomment-2024192238

Reply:
There are 2 colors, the background color and the fill color. The fill color is used when we have an icon to render on top e.g. the person shape icon. Those two colors shouldn't be the same otherwise you won't see the icon (which the regression we have now).

> The changes I made reflect this design comment https://github.com/Expensify/App/pull/38674#issuecomment-2024192238

The requested change is to change the background color which is already done (changed background color in getAvatarStyle)

Reply:
thank you for explaining, you're so right!